org.kde.trojita.json

```diff
@@ -0,0 +1,100 @@
+{
+  "app-id": "org.kde.trojita",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "base": "io.qt.qtwebkit.BaseApp",
+  "base-version": "5.15-21.08",
+  "command": "trojita",
+  "rename-desktop-file": "trojita.desktop",
+  "rename-icon": "trojita",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--persist=.abook"
```

Comment:
Is this the persistent feature from https://www.man7.org/linux/man-pages/man5/flatpak-metadata.5.html ?
Are we sure we need this one?

Reply:
Looks like Trojita need this one indeed. Not sure about the syntax however as it's the first time I see it.